go/chat/flip/dealer.go

```diff
@@ -730,7 +730,9 @@ func (d *Dealer) handleMessageStart(ctx context.Context, msg *GameMessageWrapped
 	// with our commitment. We are now in the inner loop of the Dealer, so we
 	// have to do this send in a Go-routine, so as not to deadlock the Dealer.
 	if !isLeader {
-		go d.sendCommitment(ctx, md, me)
+		if d.dh.ShouldCommit(ctx) {
```

Comment:
@mlsteele will we get warnings in the log about the fact that we weren't included in the flip but we should have been?

Reply:
See

```go
if !g.iWasIncluded {
		g.clogf(ctx, "The leader didn't include me (%s) so not sending a reveal (%s)", g.me.me, g.md)
		return nil
	}
```

in `maybeReveal`

Reply:
Yep we do get those lines. One for each `++Chat: | FlipManager: ShouldCommit -> false`. I'll look at filtering them out